### ISSUE TITLE
Compilation problem with pino-0.2.1

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#128]
```
What steps will reproduce the problem?
1. ./waf configure --prefix=/usr
2. ./waf build
3.


What is the expected output? What do you see instead?

 xxx@xxx:~/lataukset/tars/pino-0.2.1$ ./waf configure --prefix=/usr
Checking for program gcc,cc              : ok /usr/bin/gcc 
Checking for program cpp                 : ok /usr/bin/cpp 
Checking for program ar                  : ok /usr/bin/ar 
Checking for program ranlib              : ok /usr/bin/ranlib 
Checking for gcc                         : ok  
Checking for program valac               : ok /usr/local/bin/valac 
Checking for gthread-2.0                 : ok 
Checking for program version valac >= 0.1.6 : ok 0.7.10 
Checking for program msgfmt                 : ok /usr/bin/msgfmt 
Checking for program intltool-merge         : ok /usr/bin/intltool-merge 
Checking for header locale.h                : ok 
Checking for glib-2.0 >= 2.14.0             : ok 
Checking for gobject-2.0 >= 2.14.0          : ok 
Checking for gtk+-2.0 >= 2.10.0             : ok 
Checking for gee-1.0 >= 0.5.0               : ok 
Checking for gio-2.0 >= 2.0                 : ok 
Checking for libnotify                      : ok 
Checking for libsoup-2.4 >= 2.4             : ok 
Checking for libxml-2.0 >= 2.0              : ok 
Checking for webkit-1.0 >= 1.0              : ok 
Checking for unique-1.0 >= 1.0              : ok 
'configure' finished successfully (0.503s)



xxx@xxx:~/xxx/xxx/pino-0.2.1$ ./waf build
Waf: Entering directory `/xxx/xxx/xxx/xxx/pino-0.2.1/_build_'
[35/57] cc: _build_/default/src/timeline_list_abstract.c ->
_build_/default/src/timeline_list_abstract_1.o
default/src/timeline_list_abstract.c:222: error: expected declaration
specifiers or ‘...’ before ‘WebKitWebNavigationAction’
default/src/timeline_list_abstract.c:222: error: expected declaration
specifiers or ‘...’ before ‘WebKitWebPolicyDecision’
default/src/timeline_list_abstract.c:223: error: expected declaration
specifiers or ‘...’ before ‘WebKitWebNavigationAction’
default/src/timeline_list_abstract.c:223: error: expected declaration
specifiers or ‘...’ before ‘WebKitWebPolicyDecision’
default/src/timeline_list_abstract.c:298: error: expected declaration
specifiers or ‘...’ before ‘WebKitWebNavigationAction’
default/src/timeline_list_abstract.c:298: error: expected declaration
specifiers or ‘...’ before ‘WebKitWebPolicyDecision’
default/src/timeline_list_abstract.c: In function
‘_timeline_list_abstract_link_clicking_webkit_web_view_navigation_policy_decis
ion_requested’:
default/src/timeline_list_abstract.c:299: error: ‘p2’ undeclared (first use
in this function)
default/src/timeline_list_abstract.c:299: error: (Each undeclared
identifier is reported only once
default/src/timeline_list_abstract.c:299: error: for each function it
appears in.)
default/src/timeline_list_abstract.c:299: error: ‘p3’ undeclared (first use
in this function)
default/src/timeline_list_abstract.c:299: error: too many arguments to
function ‘timeline_list_abstract_link_clicking’
default/src/timeline_list_abstract.c: At top level:
default/src/timeline_list_abstract.c:721: error: expected declaration
specifiers or ‘...’ before ‘WebKitWebNavigationAction’
default/src/timeline_list_abstract.c:721: error: expected declaration
specifiers or ‘...’ before ‘WebKitWebPolicyDecision’
default/src/timeline_list_abstract.c: In function
‘timeline_list_abstract_link_clicking’:
default/src/timeline_list_abstract.c:746: error: ‘action’ undeclared (first
use in this function)
default/src/timeline_list_abstract.c:747: error: ‘decision’ undeclared
(first use in this function)
Waf: Leaving directory `/xxx/xxx/xxx/xxx/pino-0.2.1/_build_'
Build failed
 -> task failed (err #1): 
    {task: cc timeline_list_abstract.c -> timeline_list_abstract_1.o}


What version of the product are you using? On what operating system?

I am trying to compile pino-0.2.1 (I tried also with 0.2.0 and 0.1.5 but
same result). I'm using CrunchBang Linux 9.04 (which bases on Ubuntu 9.04)

Please provide any additional information below.

I have vala-0.7.10 and all the other dependencies should be fine.

```

Original issue reported on code.google.com by `hanna...@gmail.com` on 11 Mar 2010 at 10:17

**Comment by GoogleCodeExporter** [comment]
```
Hi,

What version of webkit are you using ? 
```

Original comment by `smarg...@gmail.com` on 16 Mar 2010 at 10:23

**Comment by GoogleCodeExporter** [comment]
```
Hei, my apologizes not answering earlier. Unfortunately I have since upgraded my
system which kind of provided a solution. Pino works now.
```

Original comment by `hanna...@gmail.com` on 26 Mar 2010 at 1:47

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 26 Mar 2010 at 1:52
- Changed state: **WontFix**
